BrainPortal/app/models/bourreau.rb

```diff
@@ -247,7 +247,7 @@ def self.remote_resource_info
     info.merge!(
       # Bourreau info
       :bourreau_cms              => myself.cms_class || "Unconfigured",
-      :bourreau_cms_rev          => (myself.scir_session.revision_info.to_s rescue Object.revision_info.to_s),
+      :bourreau_cms_rev          => (myself.scir_session.revision_info rescue Object.revision_info),
```

Comment:
This is wrong, the XML field cannot store a full object, the original code has a string representation of the revision info (the SVN ID string), we need something equivalent